src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/XMLMessages_de.properties

```diff
@@ -275,7 +275,7 @@
         RootElementTypeMustMatchDoctypedecl = Document Root-Element "{1}"muss mit DOCTYPE-Root "{0}" übereinstimmen.
         UndeclaredElementInContentSpec = Contentmodell des Elements "{0}" verweist auf das nicht deklarierte Element "{1}".
         UniqueNotationName = Deklaration für die Notation "{0}" ist nicht eindeutig. Ein jeweiliger Name darf nicht in mehreren Notationsdeklarationen deklariert werden.
-        ENTITYFailedInitializeGrammar =  ENTITYDatatype-Validator: Nicht erfolgreich. Initialisierungsmethode muss mit einer gültigen Grammatikreferenz aufgerufen werden. \t
```

Comment:
It looks like the _zh_CN_ file should also have the `\t` removed, but such changes are done by the translation tool. I think in this case, we should manually remove it, and then file a bug against the translation tool.

Reply:
Yeah, I believe you're right. Noted as something else to file against the translation tool.

Reply:
Just out of curiosity, do we know why `\t` was attached in the first place? I don't think it is with the original English properties.

Reply:
I can see that in 2012, the original did not have `\t` while the translated files did. (Did not search before then).
Seems like this discrepancy originates from awhile ago.